swagger-templates/php/model_doc.mustache

```diff
@@ -0,0 +1,13 @@
+{{#models}}{{#model}}# {{classname}}
+
+## Properties
+Name | Getter | Setter | Type | Description | Notes
+------------ | ------------- | ------------- | ------------- | ------------- | -------------
+{{#vars}}**{{name}}** | {{getter}}() | {{^readOnly}}{{setter}}($value){{/readOnly}} | {{#isPrimitiveType}}**{{datatype}}**{{/isPrimitiveType}}{{^isPrimitiveType}}[**{{datatype}}**]({{complexType}}.md){{/isPrimitiveType}} | {{description}} | {{^required}}[optional] {{/required}}{{#readOnly}}[readonly] {{/readOnly}}{{#defaultValue}}[default to {{{.}}}]{{/defaultValue}}
+{{/vars}}
+
+Note: All properties are protected and only accessed via getters and setters.
+
+[[Back to Model list]](../README.md#documentation-for-models) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to README]](../README.md)
```

Comment:
s/[[Back to Model list]](../README.md#documentation-for-models) /[[Back to Model list]](*../*../README.md#documentation-for-models)